Overview.bs

```diff
@@ -1723,11 +1762,11 @@ file as a patch against a source file:
 </table>
 
 
-Range Request Incremental Transfer {#range-request-incxfer}
+Binned Incremental Font Transfer (IFTB) {#binned-incxfer}
 ===========================================================
 
 <!-- TODO: remove obsolete tag once the separate range request spec is published -->
-Range request incremental font transfer is specified in a separate document: [[RangeRequest obsolete]]
+Binned Incremental Font Transfer is specified in a separate document: [[IFTB obsolete]]
```

Comment:
[[IFTB]] won't work until we've commited the IFTB document. As a workaround you can instead use a regular old link &lt;a href="IFTB.html"&gt;...&lt;/a&gt; until the document is committed.